dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```
@@ -269,3 +269,6 @@ http.headers.setIfMissing.desc=Specifies the header names and values that are se
 
 http.headers.remove=Headers to be removed from each response
 http.headers.remove.desc=Specifies the header names that are removed from each HTTP response. Any header name that is defined by using this attribute must not be empty. No header values are expected. Any header name that is defined by using this attribute must not be present in the 'add', 'set', or 'setIfMissing' header configurations.
+
+http.samesite.partitioned=Parition all listed samesite none cookies. 
+http.samesite.partitioned.desc=Specifies that the cookies listed samesite none are to be partitioned. The default value is false meaning that cookies are not partitioned. 
```

This will need some work to get the wording right. 
```suggestion
http.samesite.partitioned=Partition all outgoing SameSite=None cookies. 
http.samesite.partitioned.desc=Specifies that all SameSite=None cookies are to be partitioned and therefore sent in cross-site requests. The default value is false meaning that cookies are not partitioned. 
```

Should I ignore SameSite=None mention and just say that the cookies are partitioned? Or only mention it in the description?